### PR TITLE
fix author and reference to test pyramid article

### PR DESCRIPTION
**Pull request by marabesi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Topics already covered:
...
+If you are not familiar with testing pyramid, check out the article about it written by [Ham Vocke](https://martinfowler.com/articles/practical-test-pyramid.html) on Martin's Fowler blog. Usually, writing only unit tests for a VUE app is not enough, especially if you have a components with lots of logic and lots of dependencies. It would be better to test it together. You can try to do that in E2E tests, but E2E tests should be running against fully working and fully configured system with DB, all back-end services and without any mocks. That's very hard to achieve locally, and if you want to test all edge cases, E2E tests can take hours to run. Usually, E2E tests run after your changes were merged to master branch and were fully tested by your QA and were integrated by your CI process. It's not too late to spot the bugs here, but it delays delivery of your product because bug must be fixed first and go through PR, QA and CI again.
```
Comment (marabesi) [inline README.md]: What changed is the first paragraph only

````
If you are not familiar with testing pyramid, check out the article about it written by [Ham Vocke](https://martinfowler.com/articles/practical-test-pyramid.html) on Martin's Fowler blog
````

Comment (mcibique) [reply README.md]: ooops, that's a very good spot. my apologies for bad citation. thank you for PR.
